package com.example.bookswap

import android.os.Bundle
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import com.example.bookswap.ui.theme.BookSwapTheme
import com.example.bookswap.vista.NavegacionApp

class MainActivity : ComponentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContent {
            BookSwapTheme(darkTheme = false, dynamicColor = false) {
                NavegacionApp()
            }
        }
    }
}